patches/opencv.patch

```diff
@@ -101,13 +101,3 @@
 +            DESTINATION ${OPENCV_OTHER_INSTALL_PATH} COMPONENT libs)
    endif()
  endif()
---- catkin_ws/src/opencv3/3rdparty/libtiff/CMakeLists.txt
```

Comment:
I think the problem was that a new line is needed here.

Reply:
yes; in any case I'd rather have it separated just in case.

Reply:
Indeed, a newline was missing as expected by the previous patch. Apparently I edited by hand. Sorry for that.

Additionally I had to remove a trailing space in the last line (`# Find libm, if available`) to apply the patch without "fuzz". I would prefer to reintegrate the patch in `opencv.patch` (see [comment here](https://github.com/Intermodalics/ros_android/pull/74#discussion_r243388437)).